tasks-schema.json

```diff
@@ -259,26 +347,34 @@
                       ]
                     },
                     "quantile": {
+                      "description": "Object defining the quantiles of the predictive distribution output type.",
                       "type": "object",
                       "properties": {
                         "type_id": {
+                          "description": "Object containing required and optional arrays defining the probability levels at which quantiles of the predictive distribution will be recorded",
+                          "example": [{
+                            "required": [0.25, 0.5, 0.75],
+                            "optional": [0.1, 0.2, 0.3, 0.4, 0.6, 0.7, 0.8, 0.9]
+                          }],
```

Comment:
In the above example I included the whole `required` + `optional` structure in contrast to all other examples where I supply just a single example array. Given we are currently providing examples at higher level (i.e. not within required & optional) does this seem more appropriate for clarity?

Reply:
I think this is a helpful level of detail here.

Reply:
Great, will implement!

Reply:
Resolved in 3d6b843bfa03142fbec64b447d96c26fd02476bc